Sources/FluentKit/Properties/Siblings.swift

```diff
@@ -22,6 +22,19 @@ public final class SiblingsProperty<From, To, Through>
     var idValue: From.IDValue?
     
     public var value: [To]?
+    
+    
+    /// Allows eager loading of pivot objects through the sibling relation without adding them as children of the model.
+    /// Example:
+    ///
+    ///     Planet.query(on: db)
+    ///         .with(\.$tags).with(\.$tags.$pivots).first() { planet in
+    ///             // you can now access the loaded pivots using:
+    ///             let pivots = planet.$tags.pivots
+    ///         }
+    
```

Comment:
Let's get rid of the extra lines and the `without ...`. To be exact you'd have to say something like: "... of the `From` model". But I think it's more "to the point" to just have it like this:

```suggestion
    /// Allows eager loading of pivot objects through the sibling relation.
    /// Example:
    ///
    ///     Planet.query(on: db)
    ///         .with(\.$tags).with(\.$tags.$pivots).first() { planet in
    ///             // you can now access the loaded pivots using:
    ///             let pivots = planet.$tags.pivots
    ///         }
```